src/data/nat/up.lean

```diff
@@ -0,0 +1,58 @@
+/-
```

Comment:
Didn't you already add this file?

Reply:
It was renamed to `nat.upto`. Apparently that didn't get merged back into Simon's other work.